docs/reference/aggregations/metrics/percentile-aggregation.asciidoc

```diff
@@ -3,8 +3,8 @@
 
 A `multi-value` metrics aggregation that calculates one or more percentiles
 over numeric values extracted from the aggregated documents.  These values
-can be extracted either from specific numeric fields in the documents, or
-be generated by a provided script.
+can be extracted either from specific numeric or histogram fields in the documents,
+or be generated by a provided script.
```

Comment:
The comma is unneeded here. However, it may be more clear to swap the order of the phrases:

> These values can be generated by a provided script or extracted from
specific numeric or histogram fields in the documents.